Gemfile

```diff
@@ -78,6 +78,7 @@ gem "omniauth-rails_csrf_protection", "~> 1.0"
 
 # Doorkeeper for OAuth2
 gem "doorkeeper"
+gem "doorkeeper-openid_connect"
```

Comment:
Pretty sure rubocop is going to reject this as not correctly sorted - it will need to be moved after `doorkeeper-i18n`.